packages/components/src/internal/components/domainproperties/constants.ts

```diff
@@ -210,6 +210,14 @@ export const DEFAULT_DOMAIN_FORM_DISPLAY_OPTIONS = {
     hideConditionalFormatting: false,
 };
 
-export const DERIVATION_DATA_SCOPE_CHILD_ONLY = 'ChildOnly';
+const DERIVATION_DATA_SCOPE_CHILD_ONLY = 'ChildOnly';
+const DERIVATION_DATA_SCOPE_PARENT_ONLY = 'ParentOnly';
+const DERIVATION_DATA_SCOPE_ALL = 'All';
```

Comment:
Seems like you don't need the constants if captured in the enum, but no big deal.